uqcsbot/scripts/meme.py

```diff
@@ -0,0 +1,308 @@
+from uqcsbot import bot, Command
+import re
+from uqcsbot.util.status_reacts import loading_status, success_status
+
+API_URL = "https://memegen.link/"
+# Many different characters need to be replaced in order to work in url format
+# See the API_URL for details
+REPLACEMENTS = {'_': '__', ' ': '_', r'\"': "''", '-': '--', '?': '~q', '%': '~p', '#': '~h', '/': '~s'}
+VALID_NAMES = [
```

Comment:
Could we store the names list and meme names list as a dictionary instead so we don't have the duplicated names? Not really a blocker for merging for me though